.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+# DRILL-XXXX: Summary
```

Comment:
Could you please replace it with reference:
```suggestion
# [DRILL-XXXX](https://issues.apache.org/jira/browse/DRILL-XXXX): Summary
```

Reply:
Done!

Reply:
Thanks!